extensions/guacamole-auth-duo/pom.xml

```diff
@@ -209,19 +209,11 @@
 
     <dependencies>
 
-        <!-- Guacamole Java API -->
-        <dependency>
```

Comment:
Ah, OK.

I removed them as a part of the version number bump because of the need to check whether those numbers get bumped as well in each case. Normally, I'm an extreme proponent of "include what you use", but guacamole-common in this case is not independently used by the code; it's only used as required to interact with parts of guacamole-ext.

There's some discussion on this here: http://stackoverflow.com/questions/4226756/maven-should-i-keep-or-remove-declared-dependencies-that-are-also-transitives

IMHO: it's not a logical dependency of these extensions.